redaxo/src/addons/media_manager/lib/media_manager.php

```diff
@@ -252,9 +252,9 @@ public function sendMedia()
                 unset($header['Last-Modified']);
             }
             foreach ($header as $t => $c) {
-                header($t . ': ' . $c);
+                rex_response::setHeader($t, $c);
```

Comment:
Änderungen in dieser datei als separater PR?

Reply:
die sorgen mittels `rex_response::sendFile` dafür, dass die gecachten Dateien auch mit HTTP-Range Header funktionieren.